datasets/labelme/converter.py

```diff
@@ -0,0 +1,125 @@
+# %%
+import json
+import sys
+from pathlib import Path
+import os
+import numpy as np
+
+# %%
+if len(sys.argv) != 3:
+    sourcePath = (
+        "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/data/LabelMe/"
+    )
+    targetPath = (
+        "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/Testsyslink/"
+    )
+    answersPath = "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/data/LabelMe/answers.txt"
+    filenameTrainPath = "/home/dubar/Bureau/peerannot/fork/peerannot/datasets/labelme/data/LabelMe/filenames_train.txt"
+else:
+    sourcePath = sys.argv[1]
+    if not os.path.exists(sourcePath):
+        print("source Directory not existing")
+
+    targetPath = sys.argv[2]
+    if not os.path.exists(targetPath):
+        print("target Directory not existing")
+
+# %%
+
+
+def verifyFolders():
+    print("Coucou")
+    if os.path.exists(sourcePath + "/train"):
```

Comment:
for many concatation that follow: could you rather use
 os.path.join() 
instead of the + operator? 
that would make it windows compliant.